packages/block-library/src/classic/edit.js

```diff
@@ -54,7 +54,7 @@ export default class ClassicEdit extends Component {
 		if ( document.readyState === 'complete' ) {
```

Comment:
It seems this condition is meant to handle the case where the block is rendered after the "load" happens. So I believe right now, we might be loading the block between the `DOMContentLoaded` event and the moment the `readyState` is set to "complete".

I don't know if `load` used for the event handler is the right event to use here but we should probably be using the event that happens at the moment `document.readyState` is set to "complete" to ensure the block works properly no matter when it's rendered.

Reply:
Hm.. okay.. Thanks Riad!